src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -157,61 +124,37 @@ public PluginCompatReport testPlugins()
         }
 
         // Determine the plugin data
-        HashMap<String,String> pluginGroupIds = new HashMap<>();  // Used to track real plugin groupIds from WARs
+        Map<String, String> pluginGroupIds = new HashMap<>();  // Used to track real plugin groupIds from WARs
```

Comment:
General Java code style is to use the interface type on the left hand side of a variable declaration.